impeller/compiler/compiler.cc

```diff
@@ -155,6 +155,9 @@ static CompilerBackend CreateGLSLCompiler(const spirv_cross::ParsedIR& ir,
                              ? source_options.gles_language_version
                              : 100;
     sl_options.es = true;
+    if (source_options.type == SourceType::kFragmentShader) {
```

Comment:
TODO: verify if this makes the extension required in all shaders. If so, we'll need to add a check

Reply:
This was adding the extension to all shaders so now this is configured via an impellerc parameter